indexer/services/socks/src/lib/subscription.ts

```diff
@@ -417,6 +412,9 @@ export class Subscriptions {
 
         return resolution !== undefined;
       }
+      case (Channel.V4_PARENT_ACCOUNTS): {
```

Comment:
Why is the channel called "ACCOUNT" vs "SUBACCOUNTS"

Reply:
No reason besides old code.